### ISSUE TITLE
wrong parse of <div\n 

### ISSUE THREAD

**Issue by wvermin** [lomirus/html_editor#5]
Hi,

I like to report a bug in html_editor.
Running this program:
```
use html_editor::parse;
fn main() {
    let s = r#"
<div
> </div>
"#;

   let dom = parse(&s);
   println!("wrong: {:?}",dom);

    let s = r#"
<div> </div>
"#;
   let dom = parse(&s);
   println!("good: {:?}",dom);
}
```
I get:

```
wrong: Err("<div\n> does not match the </div>")
good: Ok([Text("\n"), Element { name: "div", attrs: [], children: [Text(" ")] }, Text("\n")])
```
So, it seems that html_editor does not handle a newline as a space or tab where it should.
The example is with &lt;div&gt;, but the bug affects also other tags like &lt;a&gt;.

Regards, and thanks for the package,

Willem

**Comment by lomirus** [comment]
Hi, Willem! Thanks for your reporting. I just fixed it in v0.5.2 and you can check it out.

**Comment by wvermin** [comment]
Hi,

thanks for your quick response!

Willem

Op zo 21 aug. 2022 16:38 schreef Lomírus ***@***.***>:
